codepropertygraph/src/test/scala/io/shiftleft/passes/CpgOverlayIntegrationTest.scala

```diff
@@ -42,15 +41,12 @@ class CpgOverlayIntegrationTest extends WordSpec with Matchers {
       // 1) add a new node
       val addNodeInverse = applyDiffAndGetInverse(cpg)(
         _.addNode(
-          new nodes.NewNode with DummyProduct {
-            override def containedNodesByLocalName = ???
-            override def label = NodeTypes.UNKNOWN
-            override def properties = Map.empty
-          }
+          nodes.NewUnknown(code = null)
         ))
       cpg.graph.nodeCount shouldBe 2
-      val additionalNode = cpg.graph.V.hasNot(NodeKeysOdb.CODE).head
+      val additionalNode = cpg.graph.V.label("UNKNOWN").head
 
+      /* we don't need to support edges with properties any longer
```

Comment:
That is not yet true. So please dont just comment this out.

Reply:
How?

This PR effectively deprecates CONTAINED_BY edges, and the only remaining use is PROPAGATE edges with the ALIAS flag in the open source dataflow tracker. However, the alias flag is always set to false, so no problem removing support for that.

Reply:
Ok, I made it true by deprecating every single edge property that is ever used. Now good?

Reply:
Resolving: We decided to do both, i.e. deprecate all existing edge properties and maintain tests and code for edge properties, on the assumption that we will need it again.